### PR TITLE
add CgroupSummary to parse /proc/cgroups

### PR DESCRIPTION
**Pull request by biello**

### CODE REVIEW

**Review by discordianfish — COMMENTED** [review]
LGTM, thanks!

File: proc_cgroups.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 The Prometheus Authors
```
Comment (SuperQ) [inline proc_cgroups.go:1]: Nit

```go
// Copyright 2021 The Prometheus Authors
```

**Review by SuperQ — COMMENTED** [review]
Minor nit, otherwise LGTM.

File: proc_cgroups.go (hunk 1 above)
Comment (biello) [reply proc_cgroups.go]: updated

**Pull request merged**